commands/sync/sync.sh

```diff
@@ -15,14 +40,35 @@ main() {
   get_arg required --type "$@"
   local type="${out:-""}"
 
-  set -x
   export RUST_LOG="${RUST_LOG:-remote-ext=debug,runtime=trace}"
 
   cargo build --release
 
   cp "./target/release/polkadot" ./polkadot-bin
 
-  ./polkadot-bin --sync="$type" --chain="$chain"
+  # Start sync.
+  # "&" runs the process in the background
+  # "> /dev/tty" redirects the output of the process to the terminal
+  nohup ./polkadot-bin --sync="$type" --chain="$chain" > "$ARTIFACTS_DIR/sync.log" 2>&1 &
```

Comment:
Any reasons for nohup here? 
If parent shell (meaning, the shell of the script) would send `HUP`, I think polkadot should receive it.

Reply:
i thought nohup is the only way to run it in a background and write output to file
may be im wrong..

Reply:
What nohup does, is starting a command, while blocking HUP signals from the parent shell. 
It's usually used for daemonizing applications, helping them to survive when the parent shell dies. 
So, a bit too much "background" for our case here.

Reply:
thanks, i've removed, re-testing